config/locales/zh_CN.yml

```
@@ -0,0 +1,40 @@
+en:
```

I think this is required to prevent overwriting all the english terms. I think this is what is causing the build to fail.

```suggestion
zh_CN:
```